Project.toml

```
@@ -84,7 +84,7 @@ PythonCall = "0.9"
 Random = "1.10"
 Random123 = "1.7"
 ReactantCore = "0.1.5"
-Reactant_jll = "0.0.83"
+Reactant_jll = "0.0.84"
```

We should probably test with the old version (without rtld_global)
```suggestion
Reactant_jll = "0.0.83"
```